vm/src/builtins/bytes.rs

```diff
@@ -128,6 +128,11 @@ impl PyBytes {
         &self.inner.elements
     }
 
+    #[pymethod(name = "__bytes__")]
+    pub fn bytes(&self) -> Vec<u8> {
+        self.inner.elements.to_vec()
+    }
+
```

Comment:
If the object itself is an exact PyBytes, we should return the reference of the object instead creating a new object.

Reply:
Is this right? 
```rust
#[pymethod(magic)]
pub fn bytes(zelf: PyRef<Self>) -> PyRef<Self> {
    zelf
}
```

Reply:
I suffered from exactly same problem in #2984. When I return PyBytes itself, below test go failed.
```python
class bytes_subclass(bytes):
    pass

bar = bytes_subclass(b'bar')
self.assertEqual(bar.__bytes__(), bar)
self.assertEqual(type(bar.__bytes__()), self.type2test)
```

Reply:
Ah I'm sorry. I understood. Let me try one more time

Reply:
@qingshi163 I applied it! Could you check one more time please? Thanks 😊